src/Gui/FreeCADGuiInit.py

```diff
@@ -291,6 +291,7 @@ def _SoGroup_init(self,*args):
 FreeCAD.addImportType("Inventor V2.1 (*.iv)","FreeCADGui")
 FreeCAD.addImportType("VRML V2.0 (*.wrl *.vrml *.wrz *.wrl.gz)","FreeCADGui")
 FreeCAD.addImportType("Python (*.py *.FCMacro *.FCScript)","FreeCADGui")
+FreeCAD.addImportType("Image formats (*.bmp *.jpg *.png *.xpm)","FreeCADGui")

```

Comment:
Can we use Qt's list of support types here, instead of hard-coding it?

Reply:
We needed to use PySide here but I don't want PySide to be a fix part of the core system. So, what we can do is: leave this code as is and after FreeCAD is fully initialized the further image types can be added.

Reply:
Removed this line and added the image types in C++